aio/deploy/recommended/kubernetes-dashboard-head.yaml

```diff
@@ -69,40 +69,64 @@ rules:
   resources: ["secrets"]
   resourceNames: ["kubernetes-dashboard-key-holder", "kubernetes-dashboard-certs", "kubernetes-dashboard-csrf"]
   verbs: ["get", "update", "delete"]
-  # Allow Dashboard to get and update 'kubernetes-dashboard-settings' config map.
+  # Allow Dashboard to get and update 'kubernetes-dashboard-head-settings' config map.
```

Comment:
The resource name below is still `kubernetes-dashboard-settings`. Same for key holder.

Reply:
Bad copy pasta/edit. Thanks.